webapp/schemas.py

```diff
@@ -21,11 +21,13 @@
     "security",
     "updates",
     "esm-infra",
+    "esm-infra-legacy",
     "esm-apps",
     "soss",
     "fips",
     "fips-updates",
     "ros-esm",
+    "realtime",
```

Comment:
We use  
`fixtures/payload`: for tests using the new pockets
`schemas`: for the api response object
`models`: the pocket options in the db

Reply:
Couldn't we have it in one place only and reference to that place from all three?

Reply:
@akbarkz  There's no duplication in the payloads, it's just making use of the new pockets so that we can test for them, as Sam said. I think there is an argument to be made for being intentional when choosing where to define types, we define status_statuses in models.py for example and the rest in (pockets, components, etc) in schemas.py where we also import status_statuses to use there as well. However changing this would require refactoring all of the columns that use these options that does not seem in scope for this pr. I can create a maintenance ticket to avoid scope creep

Reply:
@mtruj013 sounds good if you create a maintenance task. We should avoid duplication as much as possible where it can be avoided. And this seems to be a good case for that.

Reply:
Ticket created [here](https://warthogs.atlassian.net/browse/WD-14516), will merge this one as it's working as intended